src/reducers/scenes/CreateWalletReducer.ts

```diff
@@ -17,27 +17,9 @@ export interface AccountActivationPaymentInfo {
 }
 
 export interface CreateWalletState {
-  walletAccountActivationPaymentInfo: AccountActivationPaymentInfo
```

Comment:
We should also move the `AccountActivationPaymentInfo` type to the actions file.